modules/node_modules/@colony/purser-trezor/staticMethods.js

```diff
@@ -297,7 +297,7 @@ export const signMessage = async ({
   const toSign = messageOrDataValidator({ message, messageData });
   warning(messages.messageSignatureOnlyTrezor);
   try {
-    const { signature: signedMessage } = await payloadListener({
+    const { signature: signedMessage = '' } = await payloadListener({
```

Comment:
Flow showed that this was necessary; `signature` is optional in the return value, but I'm not sure whether this is a problem.